### PR TITLE
Fix Issue 211 - Reference Template Materials

### PR DESCRIPTION
**Pull request by kingthorin**
This PR closes OWASP/wstg#211.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Update CONTRIBUTING.md and README.md to include references to the template materials.

### CODE REVIEW

**Comment by kingthorin** [comment]
How's this?

Before:
image

After:
image

.... even bold doesn't stick out much, but better than before I guess.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ You can also open up a post on our [Google Group](https://groups.google.com/a/ow
     *Figure 4.8.19-1: SSTI XVWA Example*
     ```
 
+- **Please** leverage the [template materials](https://github.com/OWASP/wstg/tree/master/template).
```
Comment (victoriadrake) [inline README.md]: I'm not a fan of the floating bold. I think that placing it first in the list, directly under the "Style Guidelines" title, would be a better way to show its importance and ensure the point is read.

A further nitpicky point: not everyone may understand what "leverage" means in this context. Can we substitute a more literal instruction? Please "use/follow/refer to" the template materials, perhaps?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ This section guides you through providing new content to the testing guide. New
 - New methods to test against a certain weakness.
 - New techniques or details to test against a newly discovered weakness.
 
-In order to suggest a new scenario, follow the guidance of [how to send a PR](#how-to-send-a-PR).
+In order to suggest a new scenario, follow the guidance of [how to send a PR](#how-to-send-a-PR). **Note** you are expected to leverage the [template materials](https://github.com/OWASP/wstg/tree/master/template) when adding content.
```
Comment (victoriadrake) [inline CONTRIBUTING.md]: See other comment.

Suggestion: "When adding new content, please follow the template material."

File: README.md (hunk 1 above)
Comment (kingthorin) [reply README.md]: Yup & yup

File: CONTRIBUTING.md (hunk 2 above)
Comment (kingthorin) [reply CONTRIBUTING.md]: /me agrees

**Comment by kingthorin** [comment]
@ThunderSon updated to make the points more prominent per @victoriadrake's suggestions.

**Pull request merged**